src/NCronJob/Registry/RuntimeJobRegistry.cs

```diff
@@ -9,9 +10,16 @@ namespace NCronJob;
 public interface IRuntimeJobRegistry
 {
     /// <summary>
-    /// Gives the ability to add a job.
-    /// </summary>
-    void Register(Action<IRuntimeJobBuilder> jobBuilder);
+    /// Tries to register a job with the given configuration.
+    /// </summary>/param>
+    /// <returns>Returns <c>true</c> if the registration was successful, otherwise <c>false</c>.</returns>
+    bool TryRegister(Action<IRuntimeJobBuilder> jobBuilder);
+
+    /// <summary>
+    /// Tries to register a job with the given configuration.
+    /// </summary>/param>
+    /// <returns>Returns <c>true</c> if the registration was successful, otherwise <c>false</c>.</returns>
+    bool TryRegister(Action<IRuntimeJobBuilder> jobBuilder, [NotNullWhen(false)]out Exception? exception);
```

Comment:
Shouldn't this out param be documented through xml doc?

Reply:
Indeed - and done.